pytorch_lightning/trainer/configuration_validator.py

```diff
@@ -83,18 +81,15 @@ def __verify_train_loop_configuration(self, model):
                 ' It ensures optimizer_step or optimizer_zero_grad are called on every batch.'
             )
 
-    def __verify_eval_loop_configuration(self, model, eval_loop_name):
-        step_name = f'{eval_loop_name}_step'
-
-        # map the dataloader name
-        loader_name = f'{eval_loop_name}_dataloader'
-        if eval_loop_name == 'validation':
-            loader_name = 'val_dataloader'
+    def __verify_eval_loop_configuration(self, model):
+        stage = self.trainer._running_stage
+        step_name = f'{stage}_step'
+        loader_name = 'val_dataloader' if self.trainer.validating else f'{stage}_dataloader'
```

Comment:
```suggestion
        loader_name = f'{"val" if self.trainer.validating else "test"}_dataloader'
```

Reply:
you already suggested a `stage` variable above. `loader_name=f'{stage}_dataloader' should be enough`